src/products/use-search.tsx

```diff
@@ -26,8 +30,13 @@ export const fetcher: HookFetcher<SearchProductsData, SearchProductsInput> = (
   if (search) url.searchParams.set('search', search)
   if (Number.isInteger(categoryId))
     url.searchParams.set('category', String(categoryId))
-  if (Number.isInteger(categoryId))
-    url.searchParams.set('brand', String(brandId))
+  const categoryIds: SearchProductsInput["categoryIds"] = JSON.parse(stringifiedCategoryIds || '')
```

Comment:
Should we default this to an empty array if `stringifiedCategoryIds` is empty?

Reply:
Yes, good catch!